armips.vcxproj.filters

```diff
@@ -8,5 +8,6 @@
   <ItemGroup>
     <ClInclude Include="stdafx.h" />
     <ClInclude Include="Main\Tests.h" />
+    <ClInclude Include="Main\main.h" />
```

Comment:
Please also add all new files to CMakeLists.txt.

Reply:
Fixed in e3d89d12de83f98748e77f66f401804ad758d342